### PR TITLE
zsh-vi-mode: add head, update caveats and test

### PR DESCRIPTION
**Pull request by jeffreytse**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/zsh-vi-mode.rb [hunk 1]
```diff
@@ -14,13 +15,16 @@ def install
 
   def caveats
     <<~EOS
-      To activate the zsh vi mode, add the following at the end of your .zshrc:
+      To activate the zsh vi mode, add the following line to your .zshrc:
         source #{opt_pkgshare}/zsh-vi-mode.plugin.zsh
+
+      To check the plugin version, use the following command after activation:
```
Comment (SMillerDev) [inline Formula/zsh-vi-mode.rb:21]: This should be in the documentation, it's not homebrew specific.

Comment (jeffreytse) [reply Formula/zsh-vi-mode.rb]: Ok, just modified the caveats.

**Comment by jeffreytse** [comment]
Hi @SMillerDev , is there any problem for this PR? And now it still has not been merged.

**Comment by jeffreytse** [comment]
Hi @nandahkrishna, I don't know why such a simple PR has been processed for such a long time and there is no response. What is the problem? it just add head, update caveats, then took for a long long time. 👀

**Review by nandahkrishna — APPROVED** [review]
Thanks for the bump. It is possible for PRs to be missed, due to the large volume of PRs we receive and due to maintainers being busy.

This looks good to me, thanks @jeffreytse.

**Comment by jeffreytse** [comment]
@nandahkrishna  Thanks a lot! : D

**Pull request merged**